clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/BaseIntegrationSpec.scala

```diff
@@ -149,6 +146,23 @@ abstract class BaseIntegrationSpec(clioDescription: String)
       }, identity)
   }
 
+  /** OAuth2 credential for use in direct calls to the ClioWebClient. */
+  final implicit lazy val oauthCredential: OAuth2Credentials = {
+    AuthUtil
+      .getCredsFromServiceAccount(serviceAccount)
+      .fold(
+        fail(s"Couldn't get access token for account $serviceAccount", _),
+        identity
+      )
+  }
+
+  /**
+    * Temp file for storing service-account JSON, to pass
+    * into calls to the clio-client.
+    */
+  private lazy val serviceAccountJsonPath: Path =
```

Comment:
Even though this is type `Path` I would call it `serviceAccountJsonFile` since it's a file.